projects/packages/my-jetpack/_inc/components/welcome-flow/ConnectionStep.tsx

```diff
@@ -0,0 +1,75 @@
+import { Col, Button, Text, TermsOfService } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useCallback, useContext } from 'react';
+import { NOTICE_PRIORITY_HIGH } from '../../context/constants';
+import { NoticeContext } from '../../context/notices/noticeContext';
+import useAnalytics from '../../hooks/use-analytics';
+import styles from './style.module.scss';
+
+type ConnectionStepProps = {
+	onActivateSite: ( e?: Event ) => Promise< void >;
+	isActivating: boolean;
+};
+
+/**
+ * Component that renders the Welcome banner on My Jetpack.
+ *
+ * @param {object} props - ConnectioStepProps
+ * @param {Function} props.onActivateSite - Alias for handleRegisterSite
+ * @param {boolean} props.isActivating - Alias for siteIsRegistering
+ * @returns {object} The WelcomeBanner component.
```

Comment:
In this case, should the docblock be updated to say "`ConnectionStep` component" instead?